docs/products/opensearch/howto/opensearch-dashboard-multi_tenancy.rst

```diff
@@ -0,0 +1,76 @@
+
+Configure multi-tenancy in OpenSearch® Dashboard
+-------------------------------------------------
+
+This section provides information on configuring multi-tenancy in OpenSearch Dashboard, which involves enabling OpenSearch Security management, creating tenants, assigning roles, and mapping roles to users.
+
+Step 1: Enable OpenSearch® Security management
+```````````````````````````````````````````````
+The first step in setting up multi-tenancy in OpenSearch Dashboard is to enable OpenSearch Security management on your Aiven for OpenSearch service. Security provides authentication and authorization features that are required for multi-tenancy.  
+To enable OpenSearch Security management, see docs <link pending update>. 
+
+Step 2: Create a tenant
+`````````````````````````
+After enabling OpenSearch Security management on your Aiven for OpenSearch service, the next step is to configure tenants. A tenant is a logical grouping of users and data, each with its own set of users, roles, and permissions.
+
+OpenSearch users have access to two default tenants: Global and Private. The Global Tenant is shared by all users, and the Private Tenant is exclusively available to a single user and cannot be shared.
+
+To create a new tenant, follow these steps: 
+
+1. Log in to OpenSearch Dashboard with administrative access. 
+2. From the left navigation menu, select **Security** and select **Tenants**. 
+3. Select **Create tenant** to create a new tenant. 
+4. In the **Create Tenant** screen, enter a name and description for your new tenant.
+5. Select **Create** to save your new tenant.
+
+Step 3: Assign tenant to roles
+```````````````````````````````
+After creating a tenant, you need to assign it to a role. A role is a collection of permissions for a specific tenant that can be granted to users. 
+To assign a tenant to a role, follow these steps:
+
+1. In the OpenSearch dashboard, navigate to the **Security** section in the left-hand navigation menu, then select **Roles**. 
+2. Choose whether to create a new role or modify an existing one to include the tenant.
+3. To create a new role: 
+   
+   * Select **Create role** and enter a name for your new role.
+   * Select the permissions you want to grant to this role. 
+   * In the **Tenant permissions** section, choose the tenant you want to assign to the role from the dropdown menu. Then, select the tenant permissions for the role, such as read and/or write permissions.
+   * Select **Create** to save your new role with the assigned tenant. 
+
+4. To modify an existing role: 
+   
+   * Search for the role you want to edit and select it to view its permissions screen. 
+   * Select **Edit role** and add the required tenant in the **Tenant permissions** section. Additionally, select the tenant permissions for the role, such as read and/or write permissions.
+   * Select **Update** to save your changes.
+
+Step 4: Map roles to users
```

Comment:
shall we leave a comment here saying that user can map roles to users also via SAML integration?